KoAR.Core/ItemBuffMemory.cs

```diff
@@ -55,6 +55,8 @@ internal ItemBuffMemory(int coreOffset, int coreLength)
             }
         }
 
+        public GameSave GameSave { get; }
```

Comment:
i'd remove this property as we don't need it for this object. It never refers to the gamesave file.